third_party/move/move-compiler-v2/src/file_format_generator/function_generator.rs

```diff
@@ -897,6 +905,51 @@ impl<'a> FunctionGenerator<'a> {
         self.code.push(bc)
     }
 
+    /// Prepare a temporary, i.e., attempt to put it on the stack.
+    /// It is always safe to consider this instruction as a no-op.
+    fn prepare_arg(&mut self, ctx: &BytecodeContext, temp: TempIndex) {
+        let fun_ctx = ctx.fun_ctx;
+        let PrepareUseAnnotation(prepare_use_map) = fun_ctx
+            .fun
+            .get_annotations()
+            .get::<PrepareUseAnnotation>()
+            .expect("prepare-use annotation is a prerequisite");
+        let use_info = prepare_use_map
```

Comment:
Consider putting this logic into `PrepareUseAnnotation::get_info_at(CodeOffset)`, as other annotations do.